.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,25 +31,6 @@
 <!--- Include details of your testing environment, and the tests you ran to -->
 <!--- see how your change affects other areas of the code, etc. -->
 
-## Types of changes
```

Comment:
Maybe still keep this section?

Reply:
we talked [here](https://github.com/bentoml/BentoML/pull/1724#discussion_r661551412) about removing the section